src/test/bench_test/benchmark.cpp

```diff
@@ -152,6 +155,57 @@ void benchmark::write_random(thread_arg *thread)
     thread->stats.add_bytes(bytes);
 }
 
+void benchmark::multi_set_random(thread_arg *thread)
+{
+    if (FLAGS_benchmark_num % FLAGS_multi_count != 0) {
+        fmt::print(stderr,
+                   "num {} should be a multiple of multi_count({}).\n",
+                   FLAGS_benchmark_num,
+                   FLAGS_multi_count);
+        dsn_exit(1);
+    }
+
+    // do write operation num times
+    uint64_t bytes = 0;
+
+    for (int i = 0; i < FLAGS_benchmark_num / FLAGS_multi_count; i++) {
+        // generate hash key
+        std::string hash_key, sort_key, value;
+        hash_key = generate_string(FLAGS_hashkey_size);
```

Comment:
Define and assign hash_key in one line.
`sort_key` and `value` are not needed to declare here, you can emplace the generated strings to kvs directly, i.e. `kvs.emplace(generate_string(...), generate_string(...));`.

Reply:
It will make trouble.The argument list is built from right to left.